### ISSUE TITLE
Layout iteration - Story promo 1008+px

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2205]
**Is your feature request related to a problem? Please describe.**
We should iterate on the Story Promo for 1008+px.

**Describe the solution you'd like**
Update the Story Promo component to match with UX designs above 1008px. This should have a developer accessibility swarm.

LTR:
![story-promo-1008](https://user-images.githubusercontent.com/46446236/65327657-78db3300-dbac-11e9-84d7-659451153daf.png)

RTL:
![story-promo-1008-rtl](https://user-images.githubusercontent.com/46446236/65328052-48e05f80-dbad-11e9-81ae-b21986ce7e0e.png)

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [X] This feature is expected to need manual testing.

**Additional context**

**Comment by amywalkerdev** [comment]
Blocked until grid work is done by the Shared Components pod
